components/Dropdown/Dropdown.js

```diff
@@ -1,28 +1,28 @@
 class Dropdown {
   constructor(element) {
-    
+
     // Assign this.element to the dropdown element
-    this.element;
-    
+    this.element = document.querySelector('.dropdown');
+
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
-    
+    this.button = this.element.querySelector('.dropdown-button');
+
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
-    
+    this.content = document.querySelector('.dropdown-content');
```

Comment:
Here you will want it to be this.element.querySelector like the one above on the button